modules/engageyaBidAdapter.md

```diff
@@ -0,0 +1,68 @@
+# Overview
+
+```
+Module Name: Engageya's Bidder Adapter
+Module Type: Bidder Adapter
+Maintainer: reem@engageya.com
+```
+
+# Description
+
+Module that connects to Engageya's demand sources
+
+# Test Parameters
+```
+    var adUnits = [
+        {
+            code: 'test-div',
+            mediaTypes: {
+                banner: {
+                    sizes: [[300, 250]],  // a display size
+                }
+            },
+            bids: [
+                {
+                    bidder: "engageya",
+                    params: {
+						widgetId: '<put here widget id>',
+						websiteId: '<put here website id>',
+						pageUrl:[PAGE_URL]
```

Comment:
did you maybe think `'[PAGE_URL]'` as string here (for example), judging by L56 of your adapter, and given that from adapter code I see that this is expected to be string, not an array? (same goes for L62 of this md file)

Reply:
Yes, fixed it, will commit new version :)